src/DssSpell.t.sol

```diff
@@ -606,107 +515,35 @@ contract DssSpellTest is DssSpellTestBase {
-    function testRWA015_CONTRACT_DEPLOYMENT_SETUP() public {
-        assertEq(rwa015AJoin.vat(), addr.addr("MCD_VAT"),  "join-vat-not-match");
-        assertEq(rwa015AJoin.ilk(), "RWA015-A",            "join-ilk-not-match");
-        assertEq(rwa015AJoin.gem(), address(rwa015AGem),   "join-gem-not-match");
-        assertEq(rwa015AJoin.dec(), rwa015AGem.decimals(), "join-dec-not-match");
-
-        assertEq(rwa015AUrn.vat(),           addr.addr("MCD_VAT"),          "urn-vat-not-match");
-        assertEq(rwa015AUrn.jug(),           addr.addr("MCD_JUG"),          "urn-jug-not-match");
-        assertEq(rwa015AUrn.daiJoin(),       addr.addr("MCD_JOIN_DAI"),     "urn-daijoin-not-match");
-        assertEq(rwa015AUrn.gemJoin(),       address(rwa015AJoin),          "urn-gemjoin-not-match");
-        assertEq(rwa015AUrn.outputConduit(), address(rwa015AOutputConduit), "urn-outputconduit-not-match");
-
-        assertEq(rwa015AJar.chainlog(), addr.addr("CHANGELOG"),    "jar-chainlog-not-match");
-        assertEq(rwa015AJar.dai(),      addr.addr("MCD_DAI"),      "jar-dai-not-match");
-        assertEq(rwa015AJar.daiJoin(),  addr.addr("MCD_JOIN_DAI"), "jar-daijoin-not-match");
-
-        assertEq(rwa015AOutputConduit.dai(), addr.addr("MCD_DAI"),        "output-conduit-dai-not-match");
-        assertEq(rwa015AOutputConduit.gem(), addr.addr("USDC"),           "output-conduit-gem-not-match");
-        assertEq(rwa015AOutputConduit.psm(), addr.addr("MCD_PSM_USDC_A"), "output-conduit-psm-not-match");
-
-        assertEq(rwa015AInputConduitUrn.psm(), addr.addr("MCD_PSM_USDC_A"), "input-conduit-urn-psm-not-match");
-        assertEq(rwa015AInputConduitUrn.to(),  address(rwa015AUrn),         "input-conduit-urn-to-not-match");
-        assertEq(rwa015AInputConduitUrn.dai(), addr.addr("MCD_DAI"),        "input-conduit-urn-dai-not-match");
-        assertEq(rwa015AInputConduitUrn.gem(), addr.addr("USDC"),           "input-conduit-urn-gem-not-match");
-
-        assertEq(rwa015AInputConduitJar.psm(), addr.addr("MCD_PSM_USDC_A"), "input-conduit-jar-psm-not-match");
-        assertEq(rwa015AInputConduitJar.to(),  address(rwa015AJar),         "input-conduit-jar-to-not-match");
-        assertEq(rwa015AInputConduitJar.dai(), addr.addr("MCD_DAI"),        "input-conduit-jar-dai-not-match");
-        assertEq(rwa015AInputConduitJar.gem(), addr.addr("USDC"),           "input-conduit-jar-gem-not-match");
+    RwaLiquidationOracleLike oracle                 = RwaLiquidationOracleLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
+
+    function testRWA015_OUTPUT_CONDUIT_DEPLOYMENT_SETUP() public {
+        assertEq(rwa015AOutputConduit.dai(), addr.addr("MCD_DAI"),       "output-conduit-dai-not-match");
+        assertEq(rwa015AOutputConduit.gem(), addr.addr("PAXUSD"),        "output-conduit-gem-not-match");
+        assertEq(rwa015AOutputConduit.psm(), addr.addr("MCD_PSM_PAX_A"), "output-conduit-psm-not-match");
```

Comment:
Shouldn't we move these sanity checks back to the spell?
IIRC we just removed them there because we were hitting contract bytecode size limit.

Reply:
Yeah, I guess you're right... it makes no sense to waste gas checking a deployment when we can do it off-chain.

Reply:
It can be part of automation of checklist at some point